library/src-non-bootstrapped/scala/runtime/quoted/Matcher.scala

```diff
@@ -0,0 +1,16 @@
+package scala.runtime.quoted
+
+import scala.quoted._
+import scala.tasty._
+
+object Matcher {
```

Comment:
Should that be in `internal`? Or do we want users to access this directly?

Reply:
It should be internal. I will move it.